src/sql/workbench/contrib/notebook/browser/outputs/gridOutput.component.ts

```diff
@@ -144,8 +146,43 @@ export class GridOutputComponent extends AngularDisposable implements IMimeCompo
 			this._layoutCalledOnce = true;
 		}
 	}
+	reorderGridData(source: IDataResource) {
```

Comment:
nit - but generally for things like this that don't need to actually be class members (it doesn't need to access class vars) then I prefer to make a separate module-level function outside of the class instead